### PR TITLE
Add RocksDB package

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2268
* https://travis-ci.org/ingenue/hunter/builds/269596561?utm_source=github_status&utm_medium=notification

File: cmake/projects/RocksDB/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+hunter_add_version(
+    PACKAGE_NAME
+    RocksDB
+    VERSION
+    5.7.2-p0
+    URL
+    "https://github.com/facebook/rocksdb/archive/c41744270ac0e5ee649181211d8b09facfa45255.tar.gz"
+    SHA1
+    b470657c59f2a227fd2e115cf4bd89740266c52f
```
Comment (ruslo) [inline cmake/projects/RocksDB/hunter.cmake]: Should be `74882dfe216445be9e7c96116c3be831bb9acdab`.

**Comment by chfast** [comment]
Can you try now?

**Comment by chfast** [comment]
Documentation added.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2302
* https://travis-ci.org/ingenue/hunter/builds/270781947?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
> Testing

Some toolchains failed.

**Comment by chfast** [comment]
Is this error about spelling? https://travis-ci.org/ruslo/hunter/jobs/270419118

**Comment by ruslo** [comment]
> Is this error about spelling?

Yes, it should go away if you put name into the link section like this:
```
Added by `Paweł Bylica <https://github.com/chfast>`__ 
```

**Comment by chfast** [comment]
Done in #1231.

**Pull request closed without merging**